build-farm/platform-specific-configurations/linux.sh

```diff
@@ -135,6 +135,17 @@ if [ "${VARIANT}" == "${BUILD_VARIANT_DRAGONWELL}" ] && [ "$JAVA_FEATURE_VERSION
     fi
     export "${BOOT_JDK_VARIABLE}"="$PWD/jdk-8"
   fi
+elif [ "${VARIANT}" == "${BUILD_VARIANT_LOONGSON}" ]; then
+  if [ "$(uname -m)" = "loongarch64" ]; then
+    if [ -d /opt/j2sdk-images ]; then
+      export "${BOOT_JDK_VARIABLE}"=/opt/j2sdk-images
+    else
+      echo Loongson jdk8 requires a Loongson boot JDK - downloading one ...
+      mkdir -p "$PWD/jdk-8"
+      curl -L "http://ftp.loongnix.cn/Java/openjdk8/loongson8.1.11-jdk8u332b09-linux-loongarch64.tar.gz" | tar xpzf - --strip-components=1 -C "$PWD/jdk-8"
```

Comment:
Can we have an https download and also a checksum check afterwawrds?

Reply:
Is https available?

Reply:
OK, we'll likely have to hold off until we have that in place to make sure there's no MitM attacks on the binary blob before it gets installed on one of our agents.

Reply:
> OK, we'll likely have to hold off until we have that in place to make sure there's no MitM attacks on the binary blob before it gets installed on one of our agents.

I hava changed http to https.